### DISCUSSION TITLE
Features Request from @cds/react

### DISCUSSION THREAD

**Discussion by naminder-trantor** [vmware/clarity#6027] [Ideas]
Please add following features to @cds/react library.

Providing a reference link for the understanding of the feature requested and not found in https://clarity.design/storybook/core/?path=/docs/documentation-react--page

1. Popover :  https://getbootstrap.com/docs/5.0/components/popovers/
2. ToolTip : https://clarity.design/angular-components/tooltip/
3. Dropdown : https://clarity.design/angular-components/dropdown/
4. Navigation/Menu : https://clarity.design/angular-components/sidenav/ and https://clarity.design/angular-components/vertical-nav/
5. Notification/Toast : https://ant.design/components/notification/#header or https://getbootstrap.com/docs/5.0/components/toasts/
7. Tabs : https://clarity.design/angular-components/tab/
8. Tree : https://clarity.design/angular-components/tree-view/
9. Table : https://clarity.design/angular-components/datagrid/

**Comment by hippee-lee** [comment]
@naminder-trantor - we have a roadmap on this page: https://clarity.design/get-started/roadmap/

As far as react goes, when the Core (web-component) is published the react wrappers are also published at the same time. So the components available as web-components today are also available in the `@cds/react` package. In addition, any components that are in the roadmap will also be available as components in the `@cds/react` packages once they are merged into the `next` branch.

As you can see from the roadmap, popover, navigation, tree and table are in progress. Tree and navigation are in PR stage.
